.github/workflows/run-smc-tests.yml

```diff
@@ -6,13 +6,15 @@ on:
       - 'contracts/**'
       - 'testdata/**'
       - 'prover/**'
+      - '!**.md'
```

Comment:
I think it should be `!contracts/**/*.md`. WDYT ?

Reply:
fixed